src/Tribe/Settings_Tab.php

```diff
@@ -270,6 +296,59 @@ public function do_content(): void {
 		}
 	}
 
+	/**
+	 * Adds a child tab to the current tab.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Settings_Tab $tab The child tab to add.
+	 */
+	public function add_child( $tab ): void {
+		$this->children[ $tab->id ] = $tab;
+		$tab->parent                = $this;
+	}
+
+	/**
+	 * Checks if the current tab has children.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function has_children(): bool {
+		// Ensure all our children are tabs, as expected.
+		$this->children = array_filter(
```

Comment:
This filtering should be unnecessary with the above comment in place, and we can simplify this method to the `return` statement on line 327.